src/main/java/com/acoustic/SpringPolandSalaryCalculator/calculatorservice/AnnualGrossService.java

```diff
@@ -0,0 +1,33 @@
+package com.acoustic.SpringPolandSalaryCalculator.calculatorservice;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+
+import org.springframework.stereotype.Component;
+
+import lombok.RequiredArgsConstructor;
+
+
+@Component
+@RequiredArgsConstructor
+public class AnnualGrossService implements SalaryCalculatorService {
+
+    private static final int MONTHS_NUMBER = 12;
+
+
+    @Override
+    public int getCalculationOrder() {
+        return 6;
```

Comment:
What happens if the order changes? Do you have to open all services and change values in similar methods? Maybe it should be extracted to some configuration?

Reply:
Extracted to configuration - Done